### PR TITLE
Feat/import web

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

git-subtree-dir: packages/manager/apps/web
git-subtree-split: d34ebf62c43bc1fc97f0009c3845d68d72ce7da2

5eaeb559 — Squashed 'packages/manager/apps/web/' content from commit d34ebf62c
45087944 — Merge commit '5eaeb55973217c4c555af3498f0ec409c47f3bbf' as 'packages/manager/apps/web'
ea13b64e — Merge branch 'develop' of github.com:ovh-ux/manager into feat/import-web
94c8270a — Merge branch 'develop' of github.com:ovh-ux/manager into feat/import-web
2deec047 — Merge branch 'develop' of github.com:ovh-ux/manager into feat/import-web
bddf6df1 — build: upgrade dependencies
09d208c4 — feat: add @ovh-ux/manager-web
c013efe3 — build: upgrade yarn.lock file
fcd7f3c5 — build: upgrade dependency
001c2911 — Merge branch 'develop' of github.com:ovh-ux/manager into feat/import-web
2f0459e5 — build: update yarn.lock
d7109090 — build: add missing dependency
f6fa81f5 — merge with develop
8616340e — refactor: use @ovh-ux/manager-office & @ovh-ux/manager-sharepoint
77f1e1bb — merge with develop

ref: MANAGER-3075

### CODE REVIEW

File: packages/manager/apps/web/package.json [hunk 1]
```diff
@@ -100,6 +99,7 @@
     "filesize": "^3.5.10",
     "flatpickr": "^4.5.7",
     "font-awesome": "^4.5.0",
+    "hubspot-messenger": "HubSpot/messenger#^1.4.1",
```
Comment (FredericEspiau) [inline packages/manager/apps/web/package.json]: Sure this is still used ?

Comment (FredericEspiau) [reply packages/manager/apps/web/package.json]: https://github.hubspot.com/messenger/docs/welcome/ < I've never seen that

Comment (frenautvh) [reply packages/manager/apps/web/package.json]: yes you're right it's still used by ng-ovh-toaster but doesn't seem to be used in web anymore i'm going to remove this dep

**Comment by frenautvh** [comment]
this is an attempt to move ovh-manager-web into our mono repo, a staging is up with a working version of the manager for QC team to play with

**Comment by ovh-cds** [comment]
CDS Report sms#4173.0 ✘  
* 
  * Build CA ✔ 
  * Build EU ✘  
  * Build US ✔

**Pull request merged**